src/Containers/StoreHoursList/StoreHoursList.tsx

```diff
@@ -0,0 +1,550 @@
+    const END_INDEX_OF_SELECTION_DATE_TO_HOURS = -3;
+    const FIRST_HALF_OF_A_DAY_IN_HOURS = 12;
+    const CHECKED_INITIAL_INDEX = 0; 
+    const INITIAL_TIME_INDEX = 0; 
+    const INITIAL_DATE_INDEX = 1; 
+    const CHECKBOX_DAY = 0;
+    const CHECKBOX_TIME = 1;
+    const FIRST_TIME = 0;
+    const ALL_CATEGORIES_INDEX = 0;
+    const ALL_CATEGORIES_TIMES = 1;
+    const FIRST_CATEGORY = 0;
+
+    const [allCategoriesWithHours, setAllCategoriesWithHours] = useState<ICategoryWithHoursTypes []>(allCategories);
+    const [input, setInput] = useState('');
+    const [activeCategory, setActiveCategory] = useState(allCategoriesWithHours[FIRST_CATEGORY].category);
+    const [selectActiveCategory, setSelectActiveCategory] = useState(allCategoriesWithHours[FIRST_CATEGORY].category);
+    const [addStoreHoursCategory, setAddStoreHoursCategory] = useState(allCategoriesWithHours[FIRST_CATEGORY].category);
+    const [activeCategorySchedule, setActiveCategorySchedule] = useState<ICategoryWithHoursTypes>(allCategoriesWithHours[FIRST_CATEGORY]);
+
+    const [success, setSuccess] = useState(false);
+    const [showEmptyError, setShowEmptyError] = useState(false);
+    const [cannotDeleteActiveError, setCannotDeleteActiveError] = useState(false);
+    const [onlyOneTimeError, setOnlyOneTimeError] = useState(false);
+    const [fromTimeTooBigError, setFromTimeTooBigError] = useState(false);
+
+    const [is24, setIs24] = useState(true);
+
+    const createCategoryWithHours = (categoryName: string): ICategoryWithHoursTypes => {
+        const oneCategoryWithHours: ICategoryWithHoursTypes = {
+            category: categoryName,
+            hoursByDay: {
+                monday: [], 
+                tuesday: [], 
+                wednesday: [], 
+                thursday: [], 
+                friday: [], 
+                saturday: [], 
+                sunday: [] 
+            }
+        };
+        return oneCategoryWithHours;
+    };
+
+    const showSuccess = (content: string): React.ReactElement | null => {
+        if (success) {
+            return (
+                <Alert success> 
+                    {' '}
+                    { content }
+                    {' '}
+                </Alert> 
+            );
+        } 
+        return null;
+    };
+
+    const showError = (): React.ReactElement | null => {
+        if (showEmptyError) {
+            return (
+                <Alert error icon={CircleWithCross}> 
+                    {' '}
+                    { constants.ERRORS.ONE_ACTIVE_CATEGORY }
+                    {' '}
+                </Alert> 
+            );
+        } 
+        if (cannotDeleteActiveError) {
+            return (
+                <Alert error icon={CircleWithCross}> 
+                    {' '}
+                    { constants.ERRORS.CANNOT_DELETE_ACTIVE_CATEGORY }
+                    {' '}
+                </Alert> 
+            );
+        }
+        if (onlyOneTimeError) {
```

Comment:
These if statements should just be abstracted into a switch statement inside a function that returns alert so no need to define alert 2 times below